src/main/python/TravelTimeReporter.py

```diff
@@ -424,15 +445,17 @@ def coalesce_results(self):
         self.results = pd.DataFrame(
             {
                 "transit": self.unpivot_skim("total_transit_time"),
-                "walk": self.skims["walk_time"].query("walkTime <= @time_threshold")["walkTime"],
-                "bike": self.skims["bike_time"].query("BIKE_TIME <= @time_threshold")["BIKE_TIME"],
+                "walk": self.skims["walk_time"],
+                "bike": self.skims["bike_time"],
                 "microtransit": self.unpivot_skim("microtransit_time"),
                 "nev": self.unpivot_skim("nev_time"),
 #                "drive_alone": self.unpivot_skim("drive_alone_time")
             }
-        ).reset_index().fillna(self.settings["infinity"]).sort_values(
+        ).query(
+            "time <= @time_threshold"
```

Comment:
@JoeJimFlood is "time" a valid column name in this dataframe?

Reply:
Thank you, it's not. It's crashing at an earlier point in the code (that I'm trying to figure out) but I'll fix it before the code gets there.

Reply:
@bhargavasana I just moved the query back to where it was. It didn't fix the issue I was having anyway.